apps/docs/src/components/Preview.astro

```diff
@@ -1,14 +1,14 @@
 ---
 import { parseComponent } from "@selleo/core";
 import CodeWithCopy from "./CodeWithCopy.astro";
-const { component: Component, name, ...props } = Astro.props;
+const { component: Component, name, wrapperClasses, ...props } = Astro.props;
 const { HTMLString, ReactString } = parseComponent(Component, props);
 ---
 
 <div class="preview">
   <h3 class="font-bold leading-none text-lg">Preview - {name}</h3>
 
-  <div class="my-2">
+  <div class={`my-2 ${wrapperClasses || ""}`}>
```

Comment:
astro has something called class:list - similar to cx
https://docs.astro.build/pl/guides/styling/#combining-classes-with-classlist